cmd/antrea-controller/app/controller.go

```diff
@@ -440,6 +452,46 @@ func startNodeIPAM(client clientset.Interface,
 	return nil
 }
 
+type Authentication interface {
+	ApplyTo(authenticationInfo *genericapiserver.AuthenticationInfo, servingInfo *genericapiserver.SecureServingInfo, openAPIConfig *openapicommon.Config) error
+}
+
+type DelegatingAuthenticationOptionsWarp struct {
```

Comment:
Is it a typo of "Warp"?

Reply:
yes, updated.DelegatingAuthenticationOptionsWarp --> DelegatingAuthenticationOptionsWrap